tests/interestTest.php

```diff
@@ -17,6 +17,29 @@ class interestsTests extends TestCase {
 	 * Make sure unit tests are running.
 	 */
 	public function testNamespaceLoaded() {
-		$this->assertTrue( function_exists( '\\Pantheon\\EI\\WP\\Interest\\bootstrap' ) );
+		$this->assertTrue(
+			function_exists( '\\Pantheon\\EI\\WP\\Interest\\bootstrap' ),
+			'bootstrap function does not exist'
+		);
+	}
+
+	/**
+	 * Test script registration.
+	 */
+	public function testRegisterScript() {
+		$this->assertTrue( 
+			function_exists( '\\Pantheon\\EI\\WP\\Interest\\register_script' ),
+			'register_script function does not exist'
+		);
```

Comment:
Yeah, it's tough to test these since they don't output anything. I'd suggest some JavaScript/E2E testing to validate that they're working but that would happen on wp13n, not in this repository. You could, perhaps, validate that the functions were triggered by checking against `did_action`, e.g.: https://github.com/pantheon-systems/pantheon-wordpress-edge-integrations/blob/main/tests/geoTest.php#L288-L291

In a traditional WP Unit environment, we could test what happens when we're inside a loop and run this function (e.g. test the filters). We could still do that, although that sort of breaks the rule of "don't test WordPress doing WordPress things" and would likely be more helpful on the actual test site. 

I guess my point is, if it doesn't seem like we can test anything outside of the existence of specific functions, then the tests aren't really adding much value and should be removed in favor of E2E testing on a dev environment.

Generally, I'm not really sure what these tests are actually _testing_. Obviously they are testing if functions exist but there's no real indication of the risk of what happens if the functions _don't_ exist. Or if the code in them changes. I'm not sure they are worth keeping in the test suite if they don't give us any information about what the functions themselves are doing.